lib/guest_installation_and_configuration_base.pm

```diff
@@ -1591,6 +1625,20 @@ sub config_guest_installation_automation_registration {
     return $self;
 }
 
+# Configure guest sysinfo
+sub config_guest_sysinfo {
+    my $self = shift;
+
+    $self->reveal_myself;
+    $self->config_guest_params(@_) if (scalar(@_) gt 0);
+
+    if ($self->{guest_sysinfo} ne '') {
+        $self->{guest_sysinfo_options} .= " --sysinfo $self->{guest_sysinfo} ";
+        record_info("Guest $self->{guest_name} has been set sysinfo.");
```

Comment:
I think 
```
record_info("Guest $self->{guest_name} has been set sysinfo", "The sysinfo is $self->{guest_sysinfo}");
```
is better. Current record_info try to provide info but does not provide it completely. It does not look good.

Reply:
I really do not prefer that way -- redundant information here and there. In the dry run record_info, you can already get what are given to sysinfo.
This is very minor thing. Please just allow it being this way.